jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -73,13 +79,17 @@ def __init__(self, *args, **kwargs):
         self.spectral_unit = UnitSelectPluginComponent(self,
                                                        items='spectral_unit_items',
                                                        selected='spectral_unit_selected')
-        self.flux_unit = UnitSelectPluginComponent(self,
-                                                   items='flux_unit_items',
-                                                   selected='flux_unit_selected')
+        self.flux_or_sb_unit = UnitSelectPluginComponent(self,
+                                                         items='flux_unit_items',
+                                                         selected='flux_unit_selected')
+        self.flux_or_sb = SelectPluginComponent(self,
+                                                items='flux_or_sb_items',
+                                                selected='flux_or_sb_selected',
+                                                manual_options=['Surface Brightness', 'Flux'])
 
     @property
     def user_api(self):
-        return PluginUserApi(self, expose=('spectral_unit', 'flux_unit'))
+        return PluginUserApi(self, expose=('spectral_unit', 'flux_or_sb_unit', 'flux_or_sb'))
```

Comment:
let's only expose `flux_or_sb` for cubeviz.  Depending on the decision on my point above regarding `flux_or_sb_unit`, we may or may not want to expose that for specviz as well, but if that would require extra work beside exposing here and an if-statement in the vue file, let's create a separate ticket for flux unit conversion in specviz and imviz.

Reply:
On rebasing this PR: https://github.com/spacetelescope/jdaviz/pull/2846

spectral_unit is now only exposed. Unexposing flux_or_sb_unit is why this test is failing, I know how to use _obj/unexposed functions, I am unsure how to use this class instance if it isn't exposed. I might expose it temporarily just so I can run the CI and know if all tests are passing:
https://github.com/spacetelescope/jdaviz/blob/8b1ced75d4099e06491b7eeaee593c0081625028/jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py#L176

Reply:
This PR can re-expose whatever is needed.  We just wanted to hide them for the 3.10 release so we could have flexibility to decide on the API still.

Reply:
Bearing CI passing, I will move on to adding the `flux_unit` and `sb_unit` and toggling between them